gonum/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM clearlinux:latest AS builder
+MAINTAINER jing.j.wang@intel.com
+
+ARG swupd_args
+# Move to latest Clear Linux release to ensure
+# that the swupd command line arguments are
+# correct
+RUN swupd update --no-boot-update $swupd_args
+
+# Grab os-release info from the minimal base image so
+# that the new content matches the exact OS version
+COPY --from=clearlinux/os-core:latest /usr/lib/os-release /
+
+# Install additional content in a target directory
+# using the os version from the minimal base
+RUN source /os-release && \
+    mkdir /install_root \
+    && swupd os-install -V ${VERSION_ID} \
+    --path /install_root --statedir /swupd-state \
+    --bundles=os-core-update,go-basic,c-basic,openblas --no-boot-update
+
+# For some Host OS configuration with redirect_dir on,
+# extra data are saved on the upper layer when the same
+# file exists on different layers. To minimize docker
+# image size, remove the overlapped files before copy.
+RUN mkdir /os_core_install
+COPY --from=clearlinux/os-core:latest / /os_core_install/
+RUN cd / && \
+    find os_core_install | sed -e 's/os_core_install/install_root/' | xargs rm -d &> /dev/null || true
+
+FROM clearlinux/os-core:latest
+
+COPY --from=builder /install_root /
+ENV GOPATH /go
+ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
+ENV CGO_LDFLAGS "-lopenblas"
+
+RUN mkdir -p "$GOPATH/src" "$GOPATH/bin" && chmod -R 777 "$GOPATH"
+COPY test $GOPATH/src/test
```

Comment:
Can you use mount way to runtime mount the test code into the container instead of making it in the container image? @jwang11

Reply:
Integrate test code in the image make user easy to try gonum, however, if it is the rule, no doubt I'll follow.